subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -826,7 +828,8 @@ int nrf_cloud_agps_process(const char *buf, size_t buf_len, const int *socket)
 {
 	int err;
 	struct nrf_cloud_apgs_element element = {};
-	struct nrf_cloud_agps_system_time sys_time;
+	struct nrf_cloud_agps_system_time sys_time = {};
```

Comment:
I think empty brace init is not standard C.  Do we care about this?  I see it was already used above.

Reply:
Right, I would not normally initialize a structure to 0 this way, but the previous line was precedent.  I'm fine with changing both to `{0}`.  OK?